detection_rules/schemas/definitions.py

```diff
@@ -154,6 +154,7 @@ def validator_wrapper(value: Any) -> Any:
     "OS: Linux",
     "OS: macOS",
     "OS: Windows",
+    "Promotion: External Alerts",
```

Comment:
@peluja1012 @xcrzx any concern with extending the tag to fit our unit tests?

Reply:
Hi @Mikaayenson, the PR was merged: https://github.com/elastic/kibana/pull/228232